winterfell/webapp/src/styles/claimStart.styl

```diff
@@ -1,22 +1,89 @@
 @import "standards.styl"
 
+maxContentWidthPx = 560px
+iconWrapperPx = 54px
+
 #claim-start-view
-  width: maxLayoutWidthPx
-  margin: largeSpacing auto
-  padding: largeSpacing megaSpacing
-  box-sizing: border-box
+  min-height: 600px
```

Comment:
this was a cosmetic thing because the content itself here isn't a lot so you see the footer pretty high up.
I just chose a height big enough to make it so it's not too low (too much empty space) and not too high.